packages/projection/src/operators/certificates/withAdaHandle.ts

```diff
@@ -0,0 +1,12 @@
+import { WithCertificates } from './withCertificates';
+import { unifiedProjectorOperator } from '../utils';
+
+export interface WithAdaHandle {
+  address: string;
+  handle: string;
+}
```

Comment:
Let's generalize this and convert it to array, because block can have multiple handles:
```suggestion
export interface AddressAlias {
  address: string;
  alias: string;
  protocol: string;
}
export interface WithAddressAliases {
  addressAliases: AddressAlias[];
}
```

then we can have
```ts
export interface AdaHandle extends AddressAlias {
  address: string;
  alias: string;
  protocol: 'AdaHandle';
}
```

Reply:
The protocol should actually be `HandleStandard` 
![image](https://user-images.githubusercontent.com/303881/214677258-f90e2b38-7a48-46f2-9bcc-0df5a800474c.png)
and I think we should be able to avoid using the brand name in our codebase. The handle standard can be expressed as: Resolve the address of the asset's owner, by providing the `assetName` of a known policy. The `policyId` should be a service configuration that is included in a store of assets.